### PR TITLE
fix: add contributing guide for debugging

### PR DESCRIPTION
**Pull request by wtrocki**
As putting actual vscode file to the repo might recieve some of the pushback I have decided to document vscode debug setup.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -151,3 +151,31 @@ Reviewers, please ensure that the CLA has been signed by referring to [the contr
...
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/cmd/atlas",
+            "env": {},
+            "args": [
+              "login"
+            ]
+      },
+    ]
+}
```
Comment (andreaangiolillo) [inline CONTRIBUTING.md:179]:
```suggestion

```json
{
    "configurations": [
        {
            "name": "Login Command",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceFolder}/cmd/atlas",
            "env": {},
            "args": [
              "login"
            ]
      },
    ]
} ```

```

**Review by andreaangiolillo — APPROVED** [review]
Not blocking. Thanks for adding this

**Review by matt-condon — APPROVED** [review]
Nice one!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -151,3 +151,34 @@ Reviewers, please ensure that the CLA has been signed by referring to [the contr
...
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/cmd/atlas",
+            "env": {},
+            "args": [
+              "login"
+            ]
+      },
+    ]
+} ```
```
Comment (andreaangiolillo) [inline CONTRIBUTING.md:181]:
```suggestion
} 
```

Comment (wtrocki) [reply CONTRIBUTING.md:181]: hahaha.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -151,3 +151,34 @@ Reviewers, please ensure that the CLA has been signed by referring to [the contr
 
 For changes that involve user facing copy please include `docs-cloud-team` as a reviewer.
 
+
+## Debugging in VSCode
```
Comment (gssbzn) [inline CONTRIBUTING.md:155]: can we move this under 
Screenshot 2023-03-10 at 13 49 10
>Submitting Patches

not sure if at the same level of "Building and Testing" or under it

**Review by andreaangiolillo — APPROVED** [review]

**Comment by wtrocki** [comment]
World-class review (for initially a low-quality change). Thank you so much for all your feedback.

**Review by gssbzn — APPROVED** [review]
LGTM 💯

**Pull request merged**